sale_quotation_number/models/sale_order.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# © 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
+# © 2013 Agile Business Group sagl (<http://www.agilebg.com>)
+# © 2017 valentin vinagre  <valentin.vinagre@qubiq.es> (QubiQ)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models, api, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    keep_name_so = fields.Boolean(
+        string="Use Same Enumeration",
+        help="If this is unchecked, quotations use a different sequence from "
+             "sale orders", default=True)
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    keep_name_so = fields.Boolean(
+        related='company_id.keep_name_so',
+        readonly=False
+    )
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.multi
+    def copy(self, default=None):
```

Comment:
@manuelregidor Copying the origin seems to be confusing at times 

if Order A has an origin of "SQxxx", and it gets copied, then the new quote should not have that as it's origin.  This is in line with standard Odoo - If I duplicate an order, it does not keep on the order a reference to the order or quote it was duplicated from, so why do it here?

This is easy enough to override, but I feel it is against standard Odoo

Reply:
Hello. So do you suggest just putting in the SO's origin field the name of the SQ it comes from without including the rest of SQs and SOs it might have been coppied from? Thank you.

Reply:
@manuelregidor 

We went with:

```suggestion

    @api.multi
    @api.returns('self', lambda value: value.id)
    def copy(self, default=None):
        default = dict(default or {})
        if not 'origin' in default:
        def copy(self, default=None):
            default['origin'] = False
        return super().copy(default)
```

Just seems that it is better to put nothing than a bad accumulating origin, especially when you consider that a manually created document rarely has a meaningful "origin"

As I said, this one I am not so hung up on, as it is an easy method to write our own extension.

Reply:
Ok, I understand what you mean, but I think that's more of an improvement.